cmd/anticycle/main.go

```diff
@@ -26,18 +28,18 @@ const helpText = `Usage: anticycle [options] [directory]
   so it is ideal for searching for complex, difficult to debug cycles.
 
 Options:
-  -all               Output all packages. Default: false.
-  -format            Output format. Available: text,json. Default: text.
+  -all               Output all packages.
+  -format="text"     Output format. Available: text, json.
 
-  -exclude=""        A comma separated list of directories that should 
+  -exclude=""        A space-separated list of directories that should 
```

Comment:
Maybe it would be a good idea to use a character that is an invalid character in a directory/file name? Since for example `sample directory name` is a valid name for a directory it may lead to non-intuitive parsing. Also using `/` as the name delimiter (it is one of two characters not allowed in all of macOS/windows/*nix [the other one is `\0 (NUL)`]) would make parsing the string a lot easier.

What do you think @pawelzny?

Reply:
I thought about it. Tremendous amount of shell commands that I know, uses space as list item and arguments separator. In bash arrays are created as space-separated list of strings. If someone want to use anticycle from another shell script it will be easier to pass whole array without joining it with special character.
Then you can create script like this:

```sh
#!/bin/bash
exclude=(
    internal
    bin
    docs
)
anticycle -exclude="${exclude[*]}"
```

Of course there is no significant advantage when you use static array.
But if array of excluded folders has to be calculated it may make a difference.

Reply:
I see. I thought you were parsing it inside of go code and not using bash. Your solution is good and makes a lot of sense.